conf.py

```diff
@@ -75,7 +75,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns = ['reference/technical_information/choose_database.rst']
```

Comment:
@nidup: sphinx checks that every page is on a tree. This one is a partial, and raises a warning.